### DISCUSSION TITLE
How can I determine the version of codesandbox-client for sandpack-client's self-hosted bundler?

### DISCUSSION THREAD

**Discussion by ensorrow** [codesandbox/sandpack#59] [Q&A]
I need to self host the bundler for sandpack-client, but the master branch's code of [codesandbox-client](https://github.com/codesandbox/codesandbox-client)  doesn't match @codesandbox/sandpack-client@0.0.5. I got the error of missing browser-fs dependency for example because `yarn build:sandpack` produces no browser-fs files any more.

 How can I tell the right version (like which commit)  of  codesandbox-client for the latest sandpack-client ?

**Comment by alexnm** [comment]
I see. That's a very good point, we don't have a mechanism for this yet, but we should, since the bundler can receive regular updates for codesandbox. I'll try to document the commit sha for each release in the notes. The best thing you can do is look at the timestamp of the commits. When releasing, we run the sandpack build on the master branch of the client and we then publish to npm and generate the tag release here on this repo.

**Comment by ensorrow** [comment]
By the way, could you please fill in the details of self-hosting part document about how to host site not from the root path? I have tried the homepage field in the `package.json` file and the `CODESANDBOX_HOST` env variable, sadly nothing happened. Need more document about that, thanks a lot!

**Comment by alexnm** [comment]
@ensorrow I somehow missed your second reply, can you please give me more details here? Not sure what you are referring to exactly.
